public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -182,8 +182,9 @@ class ConfigureActions extends React.Component {
             type: toChannelType(destination.type),
             description: '',
           }));
-      } else {
-        backendErrorNotification(notifications, 'load', 'destinations', response.err);
+      } else if (response.totalMonitors !== 0) {
```

Comment:
Am hazy on my JS but are you intending to check `response.totalMonitor > 0`

if yes, why are we checking for >0 and then code comment says config index not created?